### ISSUE TITLE
Weird formating when function has only one comment inside

### ISSUE THREAD

**Issue by lucastrvsn** [JohnnyMorganz/StyLua#209]
Hello, I'm using stylua and everything is working perfect and fast! But I notice a oddly case when a function has only one comment inside and it format weird. Example:

```lua
  use {
    "~/Projects/lspplease",
    config = function()
      -- require('lspplease').setup()
    end,
  }
```

becomes:

```lua
  use {
    "~/Projects/lspplease",
    config = function()       -- require('lspplease').setup()
end,
```

https://user-images.githubusercontent.com/8826449/122781775-87c1ae80-d286-11eb-8778-6b83baa935a5.mov

**Comment by JohnnyMorganz** [comment]
Great to hear stylua is working well!
What version are you on? I believe this issue was fixed in `0.9.1`: https://github.com/JohnnyMorganz/StyLua/issues/195

**Comment by lucastrvsn** [comment]
> Great to hear stylua is working well!
> What version are you on? I believe this issue was fixed in `0.9.1`: #195

Thank you for stylua, is a amazing formatter!

I'm using 0.9.0, gonna update to 0.9.1 and try again. Thank you so much!

**Comment by lucastrvsn** [comment]
@JohnnyMorganz everything works fine. Sorry for the issue.

Thanks again!
